packages/common/src/Adaptor.js

```diff
@@ -932,3 +932,53 @@ export function assert(expression, errorMessage) {
     return state;
   };
 }
+
+/**
+ * Outputs a message to the console while maintaining the state.
```

Comment:
`while maintaining the state` seems needlessly confusing? What does maintaining state have to do with anything?

Reply:
Good question, Well the idea is your not mutating state with this function, But your right this is confusing. I will remove it. Because as long a a function takes state, you can still mutate state